udata/core/followers/models.py

```diff
@@ -15,16 +15,16 @@ class FollowQuerySet(db.BaseQuerySet):
     def following(self, user):
         return self(follower=user, until=None)
 
-    def followers(self, user):
-        return self(following=user, until=None)
+    def followers(self, followee):
```

Comment:
The debate is open: I can switch back to user. It depends on what you find clearer.
The vocabulary here is a little tricky: `user`/`followee`/`follower`/`following`.
Until then, I switched back to user but I think the `follow` module need a little bit of naming coherence in another PR.